extensions/interactions/GraphInput/directives/graph-viz.component.ts

```diff
@@ -260,6 +295,22 @@ export class GraphVizComponent implements OnInit, AfterViewInit {
       this.vizContainer[0].getScreenCTM().inverse());
     this.state.mouseX = svgp.x;
     this.state.mouseY = svgp.y;
+    if (this.state.currentMode === 2) {
```

Comment:
Maybe explain what the significance of "2" is.

Reply:
the `this.state.currentMode == 2` signifies the `add node` option is active, that is when the accessibility dot has to be shown

Reply:
Right, explain it in a comment so that other devs know.

Reply:
I have done that please take a look. thanks